doc/fr/developper/writestreamconnector.rst

```diff
@@ -0,0 +1,586 @@
+  SERVICE:mail-mars-frontend;84;memory-stats;475;0;Memory usage (Total 3.0GB): 1.19GB [buffer:0.01GB] [cache:0.59GB] [pages_tables:0.00GB] [mapped:0.00GB] [active:0.15GB] [inactive:0.04GB] [apps:0.39GB] [unused:1.81GB]
+  SERVICE:mail-jupiter-frontend;85;traffic-eth1;492;0;Traffic In : 1.41 Mb/s (1.41 %), Out : 9.08 Mb/s (9.08 %) - Total RX Bits In : 388.86 Gb, Out : 394.85 Gb
+  SERVICE:mail-jupiter-frontend;85;memory-stats;495;0;Memory usage (Total 12.0GB): 0.57GB [buffer:0.04GB] [cache:0.23GB] [pages_tables:0.02GB] [mapped:0.02GB] [active:0.07GB] [inactive:0.03GB] [apps:0.16GB] [unused:11.43GB]
+  SERVICE:mail-io-backend;88;traffic-eth1;547;0;Traffic In : 1.51 Mb/s (1.51 %), Out : 7.12 Mb/s (7.12 %) - Total RX Bits In : 389.61 Gb, Out : 390.54 Gb
+  SERVICE:mail-io-backend;88;diskio-system;551;0;Device /dev/sda: avg read 4.78 (MB/s) and write 9.08 (MB/s)
+
+Export performance data to InfluxDB
+===================================
+
+`InfluxDB <https://www.influxdata.com/>`_ is a Time Series database. We will use 
+this storage to insert performance data collected by the Centreon platform. For 
+this example, we will use the predefined `InfluxDB Docker <https://hub.docker.com/_/influxdb/>`_.
+
+To send data to InfluxDB, we need parameters to access to InfluxDB storage:
+
+* **http_server_address**: IP address of the storage
+* **http_server_port**: 8086 by default
+* **http_server_protocol**: http or https
+* **influx_database**: name of database
+* **influx_user**: user to access to database if defined
+* **influx_password**: password of user to access to database if defined
+
+In order to do not saturate the storage, we will add all events in a queue and 
+send data by bulk. We need to define the size of the queue and the maximum 
+delay before send events:
+
+* max_buffer_size  
+* max_buffer_age
+
+The LUA script will contain:
+
+A function to create the queue
+******************************
+
+This function initialises the queue with default parameters or defined by the 
+user. Moreover, the queue settings and the events will be store into a 
+dedicated structure:
+
+.. code-block:: lua
+
+  local event_queue = {
+    __internal_ts_last_flush    = nil,
+    http_server_address         = "",
+    http_server_port            = 8086,
+    http_server_protocol        = "http",
+    events                      = {},
+    influx_database             = "mydb",
+    influx_user                 = "",
+    influx_password             = "",
+    max_buffer_size             = 5000,
+    max_buffer_age              = 5
+  }
+  
+  function event_queue:new(o, conf)
+    o = o or {}
+    setmetatable(o, self)
+    self.__index = self
+    for i,v in pairs(conf) do
+      if self[i] and i ~= "events" and string.sub(i, 1, 11) ~= "__internal_" then
+        broker_log:info(1, "event_queue:new: getting parameter " .. i .. " => " .. v)
+        self[i] = v
+      else
+        broker_log:warning(1, "event_queue:new: ignoring parameter " .. i .. " => " .. v)
```

Comment:
It is the first time I see a log with a priority 1, it is interesting to make a remark about it.